src/Carousel.js

```diff
@@ -75,6 +75,9 @@ class Carousel extends Component {
         window.addEventListener('keyup', this.handleKeyboardControlUp);
         window.addEventListener('resize', this.handleResize);
 
+        // Check for presence of scrollTo in case this runs in environments
+        // where it might not exist, e.g. testing environments
+        this.containerRef.current.scrollTo && this.containerRef.current.scrollTo(0, 0);
```

Comment:
Using `scrollTo` might cause some issues in carousels with different height slides. I think this would be best as:

`this.containerRef.current && this.containerRef.current.scrollLeft = 0`

Reply:
Changed! This also removed the need for the conditional since it no longer throws while testing.